src/request.ts

```diff
@@ -238,15 +239,19 @@ export class Request {
   }
 
   public body() {
-    return this.requestParams[this.methodDescriptor.bodyAttr] as Body
+    return this.requestParams[this.methodDescriptor.bodyAttr] as Body | undefined
```

Comment:
Is this a breaking change? 🤔

Reply:
From a type perspective, maybe. But I think it was wrong, so it could be more like a bugfix: It's not certain that a request has a body, like: `Client.get({ id: 1 })` then calling request.body() and having mappersmith tell you that this is `string | Record<string, unknown>` when it is undefined is not very honest. 

The user then has a bug if they expect to get what types are telling them. So if they get this fix they will get a "breaking" bugfix, I think.

Reply:
Might be good to document it in the changeling then

Reply:
Done